tests/e2e/env/bin/get-latest-docker-tag.js

```diff
@@ -130,3 +134,13 @@ findLatestVersion( image, nameSearch ).then(
 		process.exit( 1 );
 	}
 )
+
+function getLatestMinusVersion( latestVersion, minusVersion ) {
+	// Convert the 1 or 2 to a decimal we can use for the logic below.
+	let minusAmount = minusVersion / 10;
+
+	const baseVersion = latestVersion.replace( /.[^\.]$/, '' );
+
+	// Calculate the version we need, add a patch version back in, and return.
```

Comment:
Using `WP_VERSION=5.7` will retrieve the latest bugfix release`5.7.X`. Adding the bugfix `.0` will retrieve `5.7.0`.

Also, is there a check here for the requested version being either `x.x` or `x.x.x`?

Reply:
There isn't such a check, and thanks for mentioning the `WP_VERSION` variable--I will need to test that a bit more. Thinking on this a bit more, do you think this is something we could better calculate from the Docker hub API response? The challenge I was running into was trying to figure out was knowing if there was a fix release (5.7.2 for example) so this was written to just calculate the minus pull the dot version, such as 5.7.0, instead.

Reply:
> The challenge I was running into was trying to figure out was knowing if there was a fix release

Right. My suggestion is that if you don't append the `.0` then docker will use the latest fix release for that version (eliminating the need for logic to determine if there are any).

Reply:
>My suggestion is that if you don't append the .0 then docker will use the latest fix release for that version (eliminating the need for logic to determine if there are any).

Ah! Perfect, thanks for clarifying. I'll take out that logic and look into the following check: `requested version being either x.x or x.x.x`.

Reply:
This was a great call out here @rrennick! I added in a check for the semver passed in:

```javascript
// Check if we only have a major / minor (e.g. x.x) to append a patch version
if ( latestVersion.match( /\./g ).length < 2 ) {
  latestVersion = latestVersion.concat( '.0' )
}
```

I also refactored out the script to be on its own so we can run it when `WP_VERSION` is passed in and without it: https://github.com/woocommerce/woocommerce/pull/30863/files#diff-b78dfb1597dad1c240b606316e5d557933cbf6b7e791f095e47b168b3a222c43R16